Detections/Anomalies/SignInAnomaly.yaml

```diff
@@ -0,0 +1,37 @@
+id: 9c1e9381-79dd-4ddf-9570-b73a1dc59fe0
```

Comment:
@lem0w Thank you for your contribution. Please verify from existing AAD queries (https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Azure%20Active%20Directory) if this is a duplicate. In that case, you can improve the existing query.

Reply:
Hi, thanks for the link I review the list, but wasn't able to find any similar queries.